packages/fxa-auth-server/test/remote/attached_clients_tests.js

```diff
@@ -94,11 +91,13 @@ describe('attached clients listing', function() {
     );
     allClients = await client.attachedClients();
     assert.equal(allClients.length, 2);
-    assert.equal(allClients[0].sessionTokenId, mySessionTokenId);
-    assert.equal(allClients[0].deviceId, device.id);
-    assert.equal(allClients[1].refreshTokenId, refreshTokenId);
-    assert.equal(allClients[1].deviceId, device2.id);
-    assert.equal(allClients[1].name, 'test device');
+    const one = allClients.findIndex(c => c.name === 'test device');
```

Comment:
wat?

Reply:
I was having this test fail intermittently because the order of `allClients` was reversed. idk why exactly